src/data/nav-items.yaml

```diff
@@ -134,6 +134,10 @@
       path: /experimental/chatbot/overview
     - title: Create flows
       path: /experimental/create-flows
+    - title: Delete pattern
```

Comment:
```suggestion
```
We no longer have a delete pattern, it was merged with Remove, right?

Reply:
oh yes - well spotted